net/upnp/LICENSE

```diff
@@ -0,0 +1,23 @@
+Copyright (c) 2013, John Beisley <johnbeisleyuk@gmail.com>
```

Comment:
I imagine this should be forked in a new repository, like https://github.com/tailscale/upnp, rather than including it here.

Reply:
I asked Brad about it, and I think it's probably best to keep it in the main one for now, we can fork it off later if necessary